CMakeLists.txt

```diff
@@ -85,12 +86,12 @@ target_include_directories(qserl
     ${CMAKE_CURRENT_SOURCE_DIR}/src
   )
 
-target_compile_features(qserl PRIVATE cxx_std_11)
+#target_compile_features(qserl PRIVATE cxx_std_11)
```

Comment:
Maybe we could keep the c++11 per-target feature for cmake versions that are compatible, like this:
`if(${CMAKE_MAJOR_VERSION} EQUAL 3 AND ${CMAKE_MINOR_VERSION} GREATER_EQUAL 8)`
`  target_compile_features(qserl PRIVATE cxx_std_11)`
`else()`
`  set(CMAKE_CXX_STANDARD 11)`
`endif()`
Sounds ok for you ?

Reply:
Done.